jupyter/README.MD

```diff
@@ -28,8 +28,7 @@ There are various options for customizing your Jupyter installation. These can b
 
 * `JUPYTER_PORT`=<integer>: Port on which the Jupyter server runs
 * `JUPYTER_CONDA_PACKAGES`=<colon-separated list of strings>: List of Conda packages to install.
-* `INIT_ACTIONS_REPO`=<https url>: Repo to clone to find other scripts to install/configure Conda and Jupyter.
-* `INIT_ACTIONS_BRANCH`=<string>: Branch in `INIT_ACTIONS_REPO` to use.
+* `INIT_ACTIONS_REPO`=<bucket>: GCS bucket to find other scripts to install/configure Conda and Jupyter.
```

Comment:
I think we need to do it in a backwards compatible way. Instead of replace Github with GCS, we support GCS as another type of repo, and make it default.

Reply:
Preserved backward compatibility